schema/resource.go

```diff
@@ -95,7 +95,15 @@ func (r *Resource) Columns() []string {
 	return r.Table.Columns.Names()
 }
 
-func (r *Resource) CalculateUniqueValue() error {
+func (r *Resource) CalculateUniqueValue(consistentID bool) error {
+	if !consistentID {
```

Comment:
I think this means you'll need to create a separate function for it

Reply:
I think it's a bit confusing to have a function named `CalculateUniqueValue` with a param to generate a non unique value

Reply:
Fair! I have updated the name to be `CalculateCQID`